docs/content/en/docs/install/install.md

```diff
@@ -60,14 +63,28 @@ Some helpful hints:
 * Use the `--version <version>` flag on the
   `helm upgrade --install` command to specify a different Keptn Helm chart version.
 
-* Use the following command sequence to see a list of available versions:
+  To get the appropriate chart version for the Keptn version you want,
+  use the following command sequence:
 
   ```shell
+  helm repo add keptn https://charts.lifecycle.keptn.sh
   helm repo update
-  helm search repo keptn --versions
+  helm search repo keptn
   ```
+  
+  You see that the "CHART VERSION" for `keptn/keptn` v0.9.0 is 0.3.0
+  so the following command sequence explicitly installs Keptn v0.9.0:
 
-* To verify that Keptn was installed in your cluster,
+  ```shell
+  helm repo add keptn https://charts.lifecycle.keptn.sh
+  helm repo update
+  helm upgrade --install keptn keptn/keptn \
+   --version version 0.3.0 \
+   -n keptn-lifecycle-toolkit-system --create-namespace --wait
+  ```
```

Comment:
I don't think we need this example here. We already described how to use the `--version` flag above and also had the `repo add` and `repo update` commands above already

Reply:
This was @bacherfl 's suggestion.  See my remarks below about the --values option.  Maybe we should user-test this?

Reply:
it depends on how much we want to go into detail here. I thought it might be helpful to provide an example command for using the `--version` flag, since that has been explicitly mentioned earlier, but then again we can also link to the helm docs where all the possible flags are explained in detail.

Reply:
i'm fine with keeping the examples but I think we don't need the `repo add` and `repo update` commands again in all places

Reply:
I agree, just a sentence before it we had it repeated. Since this builds on top of the previous list of commands, I think it is safe to just have here the upgrade command only

Reply:
Done